action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: 'Set this value to true to enable Azure PowerShell Login in addition to Az CLI login'
     required: false
     default: false
+  environment:
+    description: 'Name of the environment (Set this to AzureStack for AzureStack environment)'

```

Comment:
@N-Usha for the description.

Reply:
We also need to mention a default value for this parameter with value: public. Description can be "Name of the environment. Supported values are AzureStack, public. Default being public". And is the value expected to be case sensitive?

Reply:
@N-Usha I have updated the description. Also made modification to make this parameter case insensitive.